movies/src/api/index.js

```diff
@@ -0,0 +1,94 @@
+import axios from "axios";
+
+const apiKey = "51f0e9434ea11bb6ac972704b0e7f20c";
```

Comment:
Create a separate app.constants file . And declare all const there.
CONSTANTS are always declare in pascal case.